apps/squeaknode/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: yzernik/squeaknode:v0.1.21@sha256:8555988fd28b4d9e6533827510f8f618182d10de3f6e06b988d89048617615ba
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8774:8774
```

Comment:
I see the web ui is exposed at port 12994. What is port 8774 used for? Does it need to be open to the local network? If not you can remove this line.

Reply:
Ahh I just read through your docs and realised this port needs to be exposed for remote peers to connect to your. You don't need to expose this in the compose file, that's just to open it up on the local network.

We add a Tor hidden service for each app to allow remote access. Is it possible for Squeak Node to connect to other node's via Tor hidden services?

If not it should be possible by accepting a SOCKS5 proxy as part of your config and then pointing it at the Tor proxy.

Reply:
I would definitely like to use Tor hidden service for my app. That would be perfect. I don't know where to add the SOCKS5 config...